### PR TITLE
Evanhearne1 adding content to anibullying policy

### PR DESCRIPTION
**Pull request by evanhearne**
# Description

Adjusted policy.html in /pages/about/ to account for new content added to antibullying.html in /pages/about/policies/. Added edited content from Mount Sion Website Google Drive Anti Bullying Document. 

## What

Anti Bullying Section of Policies Part of Website now has content.
Anti Bullying Read More Part of Policies Part of Website now has content

## Why

To add more policies to the website. It allows more content that's useful to be added to the website.

## How

Changes were made via Sublime Text 3. Changes were made by me (evanhearne).

## Verification Steps

Clone evanhearne1-adding-content-to-antibullying-policy from evanhearne's fork
Initialize a localhost https-server in the root directory of the clone
Open localhost server and go to About --> Policies
Go down to Antibullying and ensure blurb is there
Open the Read More and ensure that document is displayed correct 
Give suggestions of changes or just approve this pull request :)

# Checklist:

- [x] This pr is linked to the related trello card
- [x] My code follows the style guidelines of this project
- [x] I have requested a code review from (at least) two team members
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This change has been verified by another team member

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@evanhearne @denispop9 I am in a meeting for the next hour. Is it possible for you guys to review each others prs.

**Comment by evanhearne** [comment]
Yes, we can review them, but we won't be able to approve them as we don't have write access.

**Comment by laurafitzgerald** [comment]
@evanhearne yes but it's good practice to do the review step as you will need to do that later and I will give you the access rights to do the review before the end of this week as it will be your responsibility after.

**Comment by evanhearne** [comment]
OK, changes have been made. @denispop9 can you please review?

**Comment by laurafitzgerald** [comment]
@denispop9 I will wait until after your review to do my review.

**Comment by evanhearne** [comment]
@denispop9 can you please review when you have the chance? :)

File: pages/about/policies/antibullying.html [hunk 1]
```diff
@@ -1 +1,401 @@
...
+<li>
+<p>The Board of Management recognises the very serious nature of bullying and the negative impact that it can have on the lives of pupils and is therefore fully committed to the following key principles of best practice in preventing and tackling bullying behaviour:</p>
+</li>
+</ol>
+<ul>
+<li>
+<p>A positive school culture and climate which</p>
+<ul>
+<li>
+<p>is welcoming of difference and diversity and is based on inclusivity</p>
```
Comment (denispop9) [inline pages/about/policies/antibullying.html]: @evanhearne The only suggestion I have is that you should tidy up the html. That html will read fine if you put the text in the 'li'

Comment (evanhearne) [reply pages/about/policies/antibullying.html]: OK, thank you, :)

Comment (evanhearne) [reply pages/about/policies/antibullying.html]: @denispop9  I'm going to leave it as it for now as it doesn't affect the functionality of the document.

Comment (laurafitzgerald) [reply pages/about/policies/antibullying.html]: @evanhearne agreed it doesn't affect the functionality but one of the items on the checklist in the description is `my code follows the style guides of this project`. As a team member @denispop9 has identified some code styling improvements you could make here.

**Comment by laurafitzgerald** [comment]
@evanhearne just reviewing the remainder of the prs. There is some outstanding feedback on this one before it can be merged. Are you okay to take a look at this?

File: pages/about/policies/antibullying.html (hunk 1 above)
Comment (evanhearne) [reply pages/about/policies/antibullying.html]: OK, @laurafitzgerald @denispop9 all changed requested have been completed.

**Pull request merged**